src/components/HeaderWallet.vue

```diff
@@ -193,9 +200,17 @@ export default {
       }
     },
 
-    disconnectWallet() {
-      this.metamaskAddress = ""
-      this.clearWallet()
+    async disconnectWallet() {
+      this.metamaskLoading = true
+      try {
+        await setEthAddress(this.api, this.pair, "")
```

Comment:
this won't work as in polkadot you cannot input empty string for the `ethAddress`. 
As discuss with @hilyds , for disconnect wallet, you just need to clear the local storage.

Reply:
I made changes according to what has been discussed, save the metamask address in local storage